src/reference/asciidoc/kafka.adoc

```diff
@@ -12,6 +14,8 @@ ListenableFuture<SendResult<K, V>> send(V data);
 
 ListenableFuture<SendResult<K, V>> send(K key, V data);
 
+ListenableFuture<SendResult<K, V>> sendKeyValue(K key, V data);
```

Comment:
One alternative thought here - instead of introducing this method, how about we rename the other two as `sendToTopic` and `sendToPartition` - maybe make overload it to take a topic as argument as well?